src/templates.ts

```diff
@@ -16,10 +17,12 @@ export function render(name: string, params?: Params): string {
   const newParams = {};
   Object.assign(newParams, params, {
     appName: 'Auth API',
+    logoUrl: getSetting('logo_url')
   });
+
   return layoutTemplate({
-    ...params,
-    body: pageTemplate(params)
+    ...newParams,
+    body: pageTemplate(newParams)
```

Comment:
In the old code, nothing was done with `newParams`, also resulting in the page's `<title>` attribute not being populated correctly.